Build broken on FreeBSD after 'move QgsCodeEditor classes to their own folder'

<!--
Bug fixing and feature development is a community responsibility, and not the responsibility of the QGIS project alone.
If this bug report or feature request is high-priority for you, we suggest engaging a QGIS developer or support organisation and financially sponsoring a fix

Checklist before submitting

- [ ] Search through existing issue reports and gis.stackexchange.com to check whether the issue already exists
- [ ] Test with a [clean new user profile](https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html?highlight=profile#working-with-user-profiles).
- [ ] Create a light and self-contained sample dataset and project file which demonstrates the issue
-->

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

On FreeBSD, I was able to build QGIS master, until [Code shuffle: move QgsCodeEditor classes to their own folder](https://github.com/qgis/QGIS/commit/1bafa2f66f7c8c5770b2250cd41f9592817a4d6f) was committed.

After the commit, the build breaks with the following error message:

```
[..snip..]
[ 46%] Building CXX object src/customwidgets/CMakeFiles/qgis_customwidgets.dir/qgsexpressionbuilderwidgetplugin.cpp.o
cd /usr/ports/graphics/qgis-devel/work/.build/src/customwidgets && /usr/bin/c++ -DPROJ_VERSION_MAJOR=6 -DQT_CONCURRENT_LIB -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_BEFORE=0x050800 -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_CAST_TO_ASCII -DQT_NO_DEBUG -DQT_PLUGIN -DQT_POSITIONING_LIB -DQT_PRINTSUPPORT_LIB -DQT_QMLMODELS_LIB -DQT_QML_LIB -DQT_QUICKWIDGETS_LIB -DQT_QUICK_LIB -DQT_SERIALPORT_LIB -DQT_SHARED -DQT_SQL_LIB -DQT_SVG_LIB -DQT_UITOOLS_LIB -DQT_WEBKITWIDGETS_LIB -DQT_WEBKIT_LIB -DQT_WIDGETS_LIB -DQT_XML_LIB -DSIP_VERSION=0x041318 -DWITH_BINDINGS -DWITH_QTWEBKIT -Dqgis_customwidgets_EXPORTS -I/usr/ports/graphics/qgis-devel/work/.build/src/customwidgets/qgis_customwidgets_autogen/include -I/usr/ports/graphics/qgis-devel/work/.build -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/raster -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/auth -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/expression -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/geometry -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/labeling -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/metadata -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/symbology -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/effects -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/core/textrenderer -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui/attributetable -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui/auth -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui/editorwidgets -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui/editorwidgets/core -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui/raster -I/usr/ports/graphics/qgis-devel/work/.build/src/core -I/usr/ports/graphics/qgis-devel/work/.build/src/core/raster -I/usr/ports/graphics/qgis-devel/work/.build/src/gui -I/usr/ports/graphics/qgis-devel/work/.build/src/customwidgets -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/external -I/usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/external/nlohmann -I/usr/ports/graphics/qgis-devel/work/.build/src/customwidgets/../ui -isystem /usr/local/include -isystem /usr/local/include/qt5 -isystem /usr/local/include/qt5/QtDesigner -isystem /usr/local/include/qt5/QtWidgets -isystem /usr/local/include/qt5/QtGui -isystem /usr/local/include/qt5/QtCore -isystem /usr/local/lib/qt5/mkspecs/freebsd-clang -isystem /usr/local/include/qt5/QtXml -isystem /usr/local/include/qt5/QtSvg -isystem /usr/local/include/qt5/QtPrintSupport -isystem /usr/local/include/qt5/QtNetwork -isystem /usr/local/include/qt5/QtWebKitWidgets -isystem /usr/local/include/qt5/QtWebKit -isystem /usr/local/include/qt5/QtSql -isystem /usr/local/include/qt5/QtConcurrent -isystem /usr/local/include/qt5/QtSerialPort -isystem /usr/local/include/qt5/QtPositioning -isystem /usr/local/include/qt5/QtDBus -isystem /usr/local/include/qt5/QtUiTools -isystem /usr/local/include/qt5/QtQuickWidgets -isystem /usr/local/include/qt5/QtQuick -isystem /usr/local/include/qt5/QtQmlModels -isystem /usr/local/include/qt5/QtQml -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -Wall -Wextra -Wno-long-long -Wformat-security -Wno-strict-aliasing -Wnon-virtual-dtor -Wno-redundant-move -Wno-return-type-c-linkage -Wno-overloaded-virtual -Wimplicit-fallthrough -Qunused-arguments -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -fPIC -fvisibility=hidden -fPIC -std=gnu++11 -o CMakeFiles/qgis_customwidgets.dir/qgsexpressionbuilderwidgetplugin.cpp.o -c /usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/customwidgets/qgsexpressionbuilderwidgetplugin.cpp
In file included from /usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/customwidgets/qgsexpressionbuilderwidgetplugin.cpp:17:
In file included from /usr/ports/graphics/qgis-devel/work/QGIS-c9fb362/src/gui/qgsexpressionbuilderwidget.h:23:
/usr/ports/graphics/qgis-devel/work/.build/src/customwidgets/../ui/ui_qgsexpressionbuilder.h:31:10: fatal error: 'qgscodeeditorexpression.h' file not found
#include "qgscodeeditorexpression.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
gmake[4]: *** [src/customwidgets/CMakeFiles/qgis_customwidgets.dir/build.make:322: src/customwidgets/CMakeFiles/qgis_customwidgets.dir/qgsexpressionbuilderwidgetplugin.cpp.o] Error 1
gmake[4]: Leaving directory '/usr/ports/graphics/qgis-devel/work/.build'
gmake[3]: *** [CMakeFiles/Makefile2:8541: src/customwidgets/CMakeFiles/qgis_customwidgets.dir/all] Error 2
gmake[3]: Leaving directory '/usr/ports/graphics/qgis-devel/work/.build'
gmake[2]: *** [Makefile:185: all] Error 2
gmake[2]: Leaving directory '/usr/ports/graphics/qgis-devel/work/.build'
*** Error code 1
```


**How to Reproduce**

It is likely that this breakage is only on FreeBSD, please read further down.

<!-- Steps, sample datasets and qgis project file to reproduce the behavior. Screencasts or screenshots welcome

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error -->

**QGIS and OS versions**

- QGIS master https://github.com/qgis/QGIS/commit/c9fb3620db8676ffa6785d7541b160b4cb90c1a3
- FreeBSD 13.0-CURRENT amd64 (r366490)
- GDAL 3.1.0
- PROJ 6.3.1

<!-- In the QGIS Help menu -> About, click in the table, Ctrl+A and then Ctrl+C. Finally paste here -->

**Additional context**
Some hours before I tried to build recent master of QGIS, the FreeBSD OS got updates for some Qt5 packages (http://svnweb.freebsd.org/changeset/ports/551499): Update py-sip to 4.19.24, qscintilla2 to 2.11.5, PyQt5 to 5.15.0 and add new py-qt5-sip 12.8.0.

The QGIS error message above contains a string about -DSIP_VERSION=0x041318, which is not the same as the installed SIP version 4.19.24.

In a Debian bug report [Bug#964132: qgis: Please switch from sip4 to sip5](https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1753208.html) they describe some problems of QGIS with finding the right SIP and PyQt5 files. [Thanks to Loïc Bartoletti for the hint]

Is there any chance, that the breakage is related to these SIP and PyQt5 updates on FreeBSD?
<!-- Add any other context about the problem here. -->

--- a/src/customwidgets/CMakeLists.txt
+++ b/src/customwidgets/CMakeLists.txt
@@ -104,6 +104,7 @@ INCLUDE_DIRECTORIES(
   ${CMAKE_SOURCE_DIR}/src/gui
   ${CMAKE_SOURCE_DIR}/src/gui/attributetable
   ${CMAKE_SOURCE_DIR}/src/gui/auth
+  ${CMAKE_SOURCE_DIR}/src/gui/codeeditors
   ${CMAKE_SOURCE_DIR}/src/gui/editorwidgets
   ${CMAKE_SOURCE_DIR}/src/gui/editorwidgets/core
   ${CMAKE_SOURCE_DIR}/src/gui/raster

